### ISSUE TITLE
BIG-IP fails to onboard when FQDN is >64 chars

### ISSUE THREAD

**Issue by memes** [F5Networks/f5-google-gdm-templates#62]
## Do you already have an issue opened with F5 support?

No.

## Description

BIG-IP fails onboarding when the `hostname` is greater than 64 characters, looking to all intents as though the BIG-IP has frozen. SSH, GUI access fail. By enabling serial access and grepping the logs the root-cause can be identified; the automatic hostname concatenated from GDM deployment name and GCP project id can easily exceed this limit.

Example: deploying to region `europe-west1` with a deployment name of `emes-gert-311-payg` will cause the deployment to fail because the generated FQDN is `bigip-1-emes-gert-311-payg.c.f5-gcs-4138-sales-cloud-sales.internal` (67 chars).

```shell
[root@localhost:Active:Standalone] google # less cloudLibsError.log {"level":"error","message":"Onboard failed: 01070903:3: Constraint 'hostname must contain less than 65 characters' failed for '/Common/system'","label":"lib/bigIpOnboard.js","timestamp":"2021-03-05T18:20:12.610Z"}
```

Destroy and redeploy same items with deployment name `emes` functions correctly as hostname is `bigip-1-emes.c.f5-gcs-4138-sales-cloud-sales.internal` (53 chars).

```shell
❯ assh admin@35.190.196.247
Warning: Permanently added '35.190.196.247' (ECDSA) to the list of known hosts.
Password:
Last login: Fri Mar  5 18:33:53 2021
admin@(bigip2-emes)(cfg-sync In Sync)(Active)(/Common)(tmos)#
```

## Template
For bugs, enter the template with which you are experiencing issues below.

via-api
via-lb
standalone

## Severity Level
For bugs, enter the bug severity level. Do not set any labels.

Severity: 2

Severity level definitions:
 1. **Severity 1 (Critical)** : Defect is causing systems to be offline and/or nonfunctional. immediate attention is required.
 2. **Severity 2 (High)** : Defect is causing major obstruction of system operations.
 3. **Severity 3 (Medium)** : Defect is causing intermittent errors in system operations.
 4. **Severity 4 (Low)** : Defect is causing infrequent interuptions in system operations.
 5. **Severity 5 (Trival)** : Defect is not causing any interuptions to system operations, but none-the-less is a bug.

**Comment by mikeshimkus** [comment]
Thanks for reporting, created internal issue ESECLDTPLT-2504 to track this.

**Comment by Goobaroo** [comment]
Ran into the same issue deploying to australia-southeastern1, I think the longest region name in GCP.

I modified the deployment to drop the .internal from the hostname, but it needs a better fix.

**Comment by leondelyon3** [comment]
Ran also in the same problem because of deployment name set to: bigip-3nic-failover-api-paygdeployment + long project name.
Found the issue by checking onboard.log:
![hostnamelongerthan65](https://user-images.githubusercontent.com/29171714/122633921-6bcdd980-d0db-11eb-8352-7c5a65b22048.png)
It will be helpful to check it directly in the python script instead of doing the full deployment then connect to the bigip instance through another vm and finally checking the log in verbose mode.

**Comment by shyawnkarim** [comment]
Closing due to age. These legacy templates are now in maintenance mode and are being replaced by our next-generation templates available in the [Cloud Templates 2.0 GitHub repo](https://github.com/F5Networks/f5-google-gdm-templates-v2).
